### PR TITLE
[JENKINS-42581] Introduce method to trigger a re-index

### PR DESCRIPTION
**Pull request by varyvol**
[JENKINS-42581](https://issues.jenkins-ci.org/browse/JENKINS-42581)

Introduce method to trigger a re-index in a Multibranch pipeline. Also take into account that a folder might not have a create view tab.

@reviewbybees @olivergondza

### CODE REVIEW

File: src/main/java/org/jenkinsci/test/acceptance/po/WorkflowMultiBranchJob.java [hunk 1]
```diff
@@ -58,4 +58,9 @@ public WorkflowJob getJob(final String name) {
         return this.getJobs().get(WorkflowJob.class, name);
     }
 
+    public void reIndex() {
+        driver.findElement(by.xpath("//div[@class=\"task\"]//*[text()=\"Scan Repository\" or text()=\"Branch Indexing\"]")).click();
+        driver.findElement(by.xpath("//div[@class=\"subtasks\"]//*[text()=\"Run Now\"]")).click();
```
Comment (jglick) [inline src/main/java/org/jenkinsci/test/acceptance/po/WorkflowMultiBranchJob.java]: https://github.com/jenkinsci/cloudbees-folder-plugin/pull/90

Comment (varyvol) [inline src/main/java/org/jenkinsci/test/acceptance/po/WorkflowMultiBranchJob.java]: @jglick thanks for the heads up. Adapted.

**Comment by varyvol** [comment]
@olivergondza please do not merge this. I needs to further investigate, but I think I need to do some changes.

**Comment by varyvol** [comment]
@olivergondza done.

File: src/main/java/org/jenkinsci/test/acceptance/po/Folder.java [hunk 2]
```diff
@@ -106,25 +104,33 @@ public String getActiveViewName() {
...
     public <T extends View> T  selectView(final Class<T> type, final String viewName) {
         final List<WebElement> viewTabs = this.getViewTabs();
+        int i = 0;
 
         for (final WebElement tab : viewTabs) {
             if (tab.getText().equals(viewName)) {
                 tab.click();
 
-                if (ALL_VIEW.equals(viewName)) {
+                if (i == 0) {
```
Comment (mafraba) [inline src/main/java/org/jenkinsci/test/acceptance/po/Folder.java]: I think a comment explaining why this is needed would be useful

Comment (varyvol) [inline src/main/java/org/jenkinsci/test/acceptance/po/Folder.java]: Added. Thanks!

**Comment by mafraba** [comment]
🐝  looks ok to me

**Comment by varyvol** [comment]
@olivergondza do you think it would be possible to get a new version released?

**Pull request merged**